g2p_seq2seq/g2p.py

```diff
@@ -169,14 +182,12 @@ def __put_into_buckets(self, source, target):
     return data_set
 
 
-  def train(self, params, train_path, valid_path, test_path):
+  def train(self, train_path, valid_path, test_path):
     """Train a gr->ph translation model using G2P data."""
 
-    if hasattr(self, 'model'):
-      print("Model already exists in", self.model_dir)
-      return
-
-    self.__train_init(params, train_path, valid_path, test_path)
+    self.__train_init(train_path, valid_path, test_path)
```

Comment:
There must be a command line option, not unconditional behavior. By default it should load the model and continue training. 

If --reinit option is specified in command line, it should reinit existing model and start training from scratch.

Command line option must be documented.